### PR TITLE
Added support for Edge browser

### PR DESCRIPTION
**Pull request by ruinianfang**
The default is still chorme, and users can choose their own browsers in login

### CODE REVIEW

**Comment by hewkick** [comment]
really good change👍

File: Login/__init__.py [hunk 1]
```diff
@@ -41,5 +41,19 @@ def login(self, **kwargs) -> None:
         """
         if self.method is None:
             raise Exception("未设置登录方式")
+        if self.method is mis:
+            #在这里选择浏览器
+            print("请选择你的浏览器")
+            print("1. Chrome")
+            print("2. Edge")
```
Comment (hewkick) [inline Login/__init__.py:48]: Maybe you should also change `config.yaml`, so that users don't need to choose the browser every time.

**Comment by ymzhang-cs** [comment]
Nice! 希望之后可以支持用配置文件选择浏览器

File: Login/__init__.py (hunk 1 above)
Comment (hewkick) [reply Login/__init__.py:48]: 卧槽发晚了，已经被merged了QWQ

Comment (ruinianfang) [reply Login/__init__.py:48]: Indeed, I forgot to make the changes, and I also forgot to modify the requirement documents. I will pay attention next time

Comment (hewkick) [reply Login/__init__.py:48]: Don't worry, I have made a PR for it, love you❤️

**Pull request merged**